### DISCUSSION TITLE
How to make a Business Website

### DISCUSSION THREAD

**Discussion by GeekNerds** [GeekNerds/Hacktoberfest-2022#241] [Q&A] [answered]
A website is essential to the business brand.

It can help you to generate leads.

It communicates information about the company and its services.

Having a website is relevant for all businesses - start building yours today!

Why make a business website
The website gives credibility to the business.

It is a good way of showing your business to potential clients and others.

In addition, to keep people updated on your company news.

The site can explain what your business is about.

The business website will help you connect with people interested in your business. Make sure that it appears in a good and presentable way!

What do I need to know to create my business website?
HTML, CSS, and JavaScript are the basic languages to create a website.

It is all you need to make your business website.

Create the structure with HTML. The first thing you have to learn, is HTML, the standard markup language for creating web pages.
Style with CSS. The next step is to learn CSS, to set the layout of your web page with beautiful colors, fonts, and much more.
Make it interactive with JavaScript. After studying HTML and CSS, you should learn JavaScript to create dynamic and interactive web pages for your users.

**Comment by keshavop** [comment, marked as answer]
A business website can also contain either following sections:
Company Vision
Available Jobs
Our Team
Our clients
Testimonials
Blog
Events
FAQ
Products
News
Terms and conditions
